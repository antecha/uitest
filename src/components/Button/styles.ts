import styled from 'styled-components';

export const StyledButton = styled.button`
  background-color: #fff;
  height: 40px;
  border: 1px solid red;
  border-radius: 4px;
`;
