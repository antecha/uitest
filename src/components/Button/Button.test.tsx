import React from 'react';
import { create } from 'react-test-renderer';
import Button from './index';

describe('Button', () => {
  const testRenderer = create(<Button />);

  it('renders', () => {
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
