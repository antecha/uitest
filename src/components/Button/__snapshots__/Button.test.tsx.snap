// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders 1`] = `
.c0 {
  background-color: #fff;
  height: 40px;
  border: 1px solid red;
  border-radius: 4px;
}

<button
  className="c0 button "
>
  Click Me again!
</button>
`;
